packages/cst-dts-gen/test/options_spec.ts

```diff
@@ -0,0 +1,84 @@
+import { generateCstDts, GenerateDtsOptions } from "../src/api"
+import { CstParser, createToken } from "chevrotain"
+import { expect } from "chai"
+
+describe("The DTS generator", () => {
```

Comment:
perhaps you can use Chai's `.to.include(` to perform less specific assertions
instead of checking equality for the whole text.
- https://www.chaijs.com/api/bdd/#method_include

That would make it easier to perform adjustments such as adding a newline argument.

Reply:
good idea